src/components/card/_card.scss

```
@@ -122,6 +127,15 @@
   }
 }
 
+.euiCard__hidden-bottom-link,
+.euiCard__hidden-top-link {
```

We stick to BEM naming conventions.
`__child` for children
`camelCase` for single names
`-isSomething` for state
`--variant` for variants

```suggestion
.euiCard__hiddenBottomLink,
.euiCard__hiddenTopLink {
```